app/build.gradle

```diff
@@ -13,6 +13,7 @@ def versionPatch = 0
 
 android {
     compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
```

Comment:
I have a question 🙌 
Android plugin for Gradle 3.0.0+ , android app project automatically uses a default version. It is better to write?

Reply:
@mhidaka 
Now, default version is `26.0.2`.
I tried with the following script.
```
task printBuildToolsVersion {
    println android.buildToolsVersion
}
```
Now, latest build tools version is `27.0.2`, is newer than the default.

Reply:
It seems good. thanks!